PAKE support (#11110)
nrfconnect/sdk-nrf: samples/crypto/ecjpake/README.rst

.. _crypto_ecjpake:

Crypto: EC J-PAKE
#################

.. contents::
   :local:
   :depth: 2

> The EC J-PAKE sample demonstrates how to do password-authenticated key exchange.

Reviewer: ```suggestion
The EC J-PAKE sample demonstrates how to do password-authenticated key exchange using the elliptic curve (EC) version of the password-authenticated key exchange by juggling (J-PAKE) protocol.
```

Reviewer: BTW, not sure if protocol is ok to use here. Or if we should use algorithm instead (as we use in the driver config table).

Requirements
************

The sample supports the following development kits:

.. table-from-sample-yaml::

Overview
********

The sample performs the following operations:

> 1. Initialization of the Platform Security Architecture (PSA) API.

Reviewer: To be in line with the other steps, change to: Initializes the Platform Security Architecture (PSA) API.

.. _crypto_ecjpake:

Crypto: EC J-PAKE
#################

.. contents::
   :local:
   :depth: 2

> The EC J-PAKE sample show how to do password-authenticated key exchange.

Reviewer 2: ```suggestion
The EC J-PAKE sample demonstrates how to do password-authenticated key exchange.
```

Requirements
************

The sample supports the following development kits:

.. table-from-sample-yaml::

Overview
********

The sample performs the following operations:

1. Initialization of the Platform Security Architecture (PSA) API.

> 

Reviewer 2: ```suggestion
```
Think this empty line needs to come out for the list to render correctly. Sorry, I think this slipped in through my previous suggestion.

> #. Goes through the steps for J-PAKE on server and client side and verifies

Reviewer 2: ```suggestion
#. Goes through the steps for J-PAKE on server and client sides.
#. Verifies that the derived keys are the same.
```

>    that the derived keys are the same.

Reviewer 2: ```suggestion
```
